jupyterhub/spawners.py

```diff
@@ -325,24 +325,23 @@ def get_pod_manifest(self):
                 image='alpine/git:latest',
                 command=['sh', '-c'],
                 args=[
-                    'export GIT_LFS_SKIP_SMUDGE={lfs_skip_smudge} && '
                     'rm -rf {mount_path}/* && '
                     '(rm -rf {mount_path}/.* || true) && '
                     'apk update && apk add git-lfs && '
+                    'git lfs --install {lfs_skip_smudge} && '
```

Comment:
`--local` :scream:

Reply:
not needed anymore: https://github.com/SwissDataScienceCenter/renku-python/pull/347/files#diff-3d81f436074c1bb23a072c4663d07c18R363

Reply:
Do we wipe all existing projects then? :wink:

Reply:
so for backwards compatibility we need this line, plus a line with `--local` after the repo is cloned. 😭